src/bees.cc

```diff
@@ -622,6 +628,12 @@ bees_main(int argc, char *argv[])
 			case 't':
 				chatter_prefix_timestamp = true;
 				break;
+			case 'P':
+				crucible::set_relative_path(cwd);
+				break;
+			case 'p':
+				crucible::set_relative_path("");
+				break;
 			case 'h':
 				do_cmd_help(argv);
 			default:
```

Comment:
bees.cc: In function ‘int bees_main(int, char**)’:
bees.cc:626:16: error: this statement may fall through [-Werror=implicit-fallthrough=]
     do_cmd_help(argv);
     ~~~~~~~~~~~^~~~~~
bees.cc:627:4: note: here
    default:
    ^~~~~~~
- - -
~ gcc --version
gcc (GCC) 7.2.0
Copyright (C) 2017 Free Software Foundation, Inc.
This is free software; see the source for copying conditions.  There is NO
warranty; not even for MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.

Reply:
Why does your compiler have that warning turned on? It's not the same here...